### DISCUSSION TITLE
Tests

### DISCUSSION THREAD

**Discussion by peternepp** [fabiangeisler/fg-tools#4] [General]
I'm a bit confused with the [test setup](https://github.com/fabiangeisler/fg-tools/tree/master/tests). Some tests seems to be obsolete and the instructions made in [mayaUnittest.py](https://github.com/fabiangeisler/fg-tools/blob/master/tests/mayaUnittest.py) doesn't make any sense to me. Did I miss something?

**Comment by fabiangeisler** [comment]
Unfortunately the test setup did never work in the fg-tools. I had plans to write some tests but then live intervened. 🙈 Guess a lot if can go. I found this automatic Maya test setup https://github.com/mischakolbe/node_calculator/blob/main/.github/workflows/noca_unittests.yml which looks really promising. But I'd still have to write tests 😬 . Feel free to add/remove/change.

**Reply by peternepp** [reply]
When the test setup isn't working actually, maybe removing the **tests** folderto help avoiding confusion in the first place?

**Reply by fabiangeisler** [reply]
Indeed. Maybe a good first issue! 😄 Will add that.
